frontend/src/components/shared/Radio/Radio.tsx

```diff
@@ -81,16 +81,14 @@ class Radio extends React.PureComponent<Props, State> {
 
     return (
       <div className="row-widget stRadio" style={style}>
-        {label || help ? (
-          <StyledWidgetLabel>
-            {label}
-            {help && (
-              <StyledWidgetLabelHelpInline>
-                <TooltipIcon content={help} placement={Placement.TOP_RIGHT} />
-              </StyledWidgetLabelHelpInline>
-            )}
-          </StyledWidgetLabel>
-        ) : null}
+        <WidgetLabel visible={!!label}>
```

Comment:
I removed `help` from here. No other widget did this, really, and it can only impact our own usage of `Radio` anyway since `label` is never falsy when coming from the Python side. 

Plus we don't actually use Radio ourselves, and if we ever do we shouldn't have the combination of no label + help, as that would be visually strange.